Jenkinsfile-core-dev

```diff
@@ -0,0 +1,59 @@
+pipeline {
+    agent {
+        label 'ec2-jdk8'
+    }
+
+    triggers {
+        pollSCM('H/30 * * * *')
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+    }
+
+    environment {
+        MAVEN_OPTS = '-Xms1024m -Xmx4096m'
+    }
+
+    stages {
+        stage ('Build') {
+            steps {
+                echo 'Building DHIS2 ...'
+                script {
+                    withMaven {
+                        sh 'mvn -T 4 clean install -f dhis-2/pom-full.xml -Pjdk8 --update-snapshots'
+                    }
+                }
+            }
+        }
+
+        stage ('Sync WAR') {
+            steps {
+                echo 'Syncing WAR ...'
+                sh 'curl "https://raw.githubusercontent.com/dhis2/dhis2-server-setup/master/ci/scripts/sync-war-s3.sh" -o sync-war-s3.sh'
+                sh 'chmod +x sync-war-s3.sh'
+                sh './sync-war-s3.sh ${GIT_BRANCH}'
```

Comment:
how will this work for patch/x branches that we use for rc? or is that going to be a separate pipeline?

Reply:
This will only work for the "*-dev" builds, i.e. the `2.36` - `2.34` branches + `master` for now. We'll probably have a separate pipeline for the RC and other types of builds we need.